.gitlab/spack_packages/sundials/package.py

```diff
@@ -169,6 +178,9 @@ class Sundials(CachedCMakePackage, CudaPackage, ROCmPackage):
         "profiling", default=False, when="@6.0.0:", description="Build with profiling capabilities"
     )
 
+    # Scheduler
+    variant("scheduler", default="slurm", description="Specify which scheduler the system runs on.", values=("flux", "lsf", "slurm"))
```

Comment:
Looks like only the `flux` case is handled below. Should we be using `jsrun` and `srun` for LSF and SLURM, respectively? Should this have a "mpiexec" or "none" option to use whatever CMake finds for the MPI executable?

Reply:
I can add both to the variant, and define jsrun and srun for the two schedulers, though I think the system currently defaults to mpiexec and srun for the two currently, but it'd be good to be explicit about that.